README.md

```diff
@@ -183,7 +183,7 @@ $q = new Queue(10, null, function ($url) use ($browser) {
 $promise = $q($url);
 ```
 
-The `$q` instance is invokable, so that invoking `$q(...$args)` will
+The `$q` instance is invocable, so that invoking `$q(...$args)` will
```

Comment:
I'm not sure about this one, feels like both would be valid variants, no? Couldn't find a definitive source, but perhaps you have more insights? It looks like "invokable" is used more often than "invocable" in a programming context?

Reply:
Was researching the topic and it seems like you are definitely right with that one. It was also said in #44 as I saw. Thank you for this information. 🌟